src/main/drivers/vtx_rtc6705.c

```diff
@@ -170,40 +183,77 @@ static void rtc6705Transfer(uint32_t command)
     spiTransferByte(RTC6705_SPI_INSTANCE, (command >> 8) & 0xFF);
     spiTransferByte(RTC6705_SPI_INSTANCE, (command >> 0) & 0xFF);
 
+    delayMicroseconds(2);
+
     DISABLE_RTC6705;
+
+    delayMicroseconds(2);
 }
 
 /**
  * Set a band and channel
  */
-void rtc6705SetChannel(uint8_t band, uint8_t channel)
+void rtc6705SetChannel(const uint8_t band, const uint8_t channel)
 {
-    band = constrain(band, RTC6705_BAND_MIN, RTC6705_BAND_MAX);
-    channel = constrain(channel, RTC6705_CHANNEL_MIN, RTC6705_CHANNEL_MAX);
+    uint8_t constrainedBand = constrain(band, 0, RTC6705_BAND_COUNT - 1);
```

Comment:
I'll revert the const usage in the method signature for the pass by value arguments.